docs/DeveloperGuide.md

```diff
@@ -236,13 +236,12 @@ Additionally, the classes implement the following operations:
 
 Given below is an example usage scenario and how the grade progress feature behaves at each step.
 
-Step 1. The user launches the application for the first time. The `AddressBook` will be initialized with the sample data and
-each person in the `AddressBook` contains an empty `GradeProgressList` object.
+Step 1. The user launches the application for the first time. The `AddressBook` will be initialized with the sample data.
+
+Step 2. The user uses `view Roy Balakhrisnan` who is a `Person` without any grade progress.
```

Comment:
not sure why this is necessary, especially since this command doesn't have to be used in full view mode, also cos this command can be used regardless if the grade progress is empty or not

Reply:
yourre right done!